File: beginner/c/1150.c
#include <stdio.h>
int main(){

    int x, z, i, soma = 0, cont = 0;

    scanf("%d", &x);
    scanf("%d", &z);

    while (z <= x){
        scanf("%d", &z);
    }

    for(;;){
        if(soma > z){
            break;
        }else{
            cont++;
            soma += x;
            x++;
        }
    }

    printf("%d\n", cont);
    

    return 0;
}

File: beginner/javascript/1150.js
var input = require('fs').readFileSync('stdin', 'utf8');
var valores = input.split('\n');

var x, z, i, soma = 0, cont = 0;

x = parseInt(valores.shift());
z = parseInt(valores.shift());

while (z <= x){
    z = parseInt(valores.shift());
}

for(;;){
    if(soma > z){
        break;
    }else{
        cont++;
        soma += x;
        x++;
    }
}

console.log(cont);
